addons/account_voucher/migrations/9.0.1.0/post-migration.py

```diff
@@ -113,16 +111,31 @@ def create_voucher_line_tax_lines(env):
         "where v.tax_id is not null")
 
 
+@openupgrade.logging()
+def set_voucher_line_amount(env):
+    """We replicate here the code of the compute function and set the value
+    finally via SQL for avoiding the trigger of the rest of the computed
+    fields that depends on this field.
+    """
+    lines = env['account.voucher.line'].search([])
+    for line in lines:
```

Comment:
Can you apply `openupgrade.chunked(lines)` to prevent MemoryErrors and improve performance on big installations?

Reply:
Done